src/ontoweaver/tabular.py

```diff
@@ -870,9 +899,14 @@ def __call__(self):
                         if extracted_metadata:
                             metadata.update(extracted_metadata)
 
+        # Extract input data validation schema from yaml file and instantiate a Pandera DataFrameSchema object and validator.
+        validation_rules = self.get(k_validate)
+        yaml_validation_rules = yaml.dump(validation_rules, default_flow_style=False)
+        validation_schema = pa.DataFrameSchema.from_yaml(yaml_validation_rules)
+
         logging.debug(f"source class: {source_t}")
         logging.debug(f"properties_of: {properties_of}")
         logging.debug(f"transformers: {transformers}")
         logging.debug(f"metadata: {metadata}")
-        return subject_transformer, transformers, metadata
+        return subject_transformer, transformers, metadata, validate.InputValidator(validation_schema)
```

Comment:
If the validation call can fail, it would be better to call it in a try block that would log it somehow.